_posts/2018-05-14-coding-period-begins.md

```diff
@@ -0,0 +1,42 @@
+---
+    date: 2018-05-14
+    category: Bonding Period
+---
+
+##### Bonding Period Ends
+26 April 2018 - 13 May 2018
+
+<br/><br/>
+Now that the bonding period has finally come to an end, I thought
+I should let you know about the developments during the period and
+my plan ahead. Bonding Period is the time meant to help the students
+get started with the community, coding practices and discuss the details
+of the project one plans to follow with the community and their mentors.
+I had a great time knowing the community and the interesting stuff they do.
+
+<br/><br/>
+> ##### Developments
+
+1. **Resolved pending PR's**: 
+  I joined the coala community in late March and made a few 
+  contributions to projects.coala.io and coala-vs-code of 
+  which some were left pending in review. I got all of them 
+  resolved and merged.
+
+2. <br/>**Finish newcomer process**: 
+  Since the deadline was too close when I joined the community 
+  I was instrcuted to work on my proposal first and worry with 
+  the newcomer process later on. I have since completed the 
+  newcomer process.<br/>
+
+3. <br/>**Schedule**: 
+  I worked with my primary(@gaocegege) and we have decided on a week 
```

Comment:
I think the template does not support GitHub flavoured markdown. Then we should use `[@gaocegege](https://github.com/gaocegege)`